Code/Sideloaded-TextTracks-Examples/Cocoapod/SideloadedTextTracksExample/PlayerView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PlayerView.swift
+//  SideloadedTextTracksExample
+//
+//  Created by Wonne Joosen on 13/04/2024.
+//
+
+import UIKit
+import THEOplayerSDK
+
+/// A container UIView that holds a `THEOPlayer` and lays it out
+/// to stretch the entire frame of this container view
+class PlayerView: UIView {
+    /// The player that is streched out to fit this view
+    let player: THEOplayer
+    
+    required init?(coder: NSCoder) {nil}
```

Comment:
You can always call `super.init` instead of returning `nil`.

Reply:
left this as is, just like the other example apps in this repo